message_sender/tasks.py

```diff
@@ -77,6 +79,44 @@ def run(self, metric_name, metric_value, session=None, **kwargs):
 fire_metric = FireMetric()
 
 
+class Concurrency_Limiter(object):
+    def __init__(self):
+        self.redis_server = redis.StrictRedis(host=settings.REDIS_HOST,
+                                              port=settings.REDIS_PORT,
+                                              db=settings.REDIS_DB)
```

Comment:
Yes, we've used it for the metrics value cache: 

https://github.com/praekelt/hellomama-registration/blob/develop/registrations/models.py#L174-L186
https://github.com/praekelt/hellomama-registration/blob/develop/hellomama_registration/settings.py#L221-L226
https://docs.djangoproject.com/en/1.10/topics/cache/